components/message.php

```diff
@@ -0,0 +1,504 @@
+<?php
+/**
+ * Manage BuddyPress Messages.
+ *
+ * @since 1.6.0
+ */
+class BPCLI_Message extends BPCLI_Component {
+
+	/**
+	 * Object fields.
+	 *
+	 * @var array
+	 */
+	protected $obj_fields = array(
+		'id',
+		'subject',
+		'message',
+		'thread_id',
+		'sender_id',
+		'date_sent',
+	);
+
+	/**
+	 * Add a message.
+	 *
+	 * ## OPTIONS
+	 *
+	 * --from=<user>
+	 * : Identifier for the user. Accepts either a user_login or a numeric ID.
+	 *
+	 * [--to=<user>]
+	 * : Identifier for the recipient. Accepts either a user_login or a numeric ID.
+	 * ---
+	 * Default: Empty.
+	 * ---
+	 *
+	 * [--subject=<subject>]
```

Comment:
I'd argue that `subject` and `content` should be required. The default values are potentially useful for generating random messages, but in that case they should go into the `generate()` method rather than here. There are really no real-life situations where you'd want to create a message with random text. Empty fields could still be supported with `--content=''` or `--subject=''`. What do you think?